packages/react-core/src/components/DataList/examples/DataListDraggable.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import {
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListCheck,
+  DataListControl,
+  DataListDragButton,
+  DataListItemCells,
+  DragDrop,
+  Draggable,
+  Droppable
+} from '@patternfly/react-core';
+
+interface ItemType {
+  id: string;
+  content: string;
+}
+
+const getItems = count =>
+  Array.from({ length: count }, (v, k) => k).map(k => ({
+    id: `item-${k}`,
+    content: `item ${k} `.repeat(k === 4 ? 20 : 1)
```

Comment:
Nitpick: may be something worth fixing in another issue, but I think it'd be worth giving these `v` and `k` params more meaningful names